.github/workflows/main.yml

```diff
@@ -60,8 +83,37 @@ jobs:
       - name: Create working directory
         run: mkdir /tmp/bundle
 
+      # generate the bundle using operator-sdk-0.19.4
       - name: Generate bundle
         run: WORKING_DIR=/tmp/bundle ./build/generate_bundle.sh
 
+      # prepare the environment to run bundle validation and bundle scorecard checks
+      - name: Get operator-sdk image 1.26.0
+        run: curl --output operator-sdk-$RELEASE_VERSION -JL https://github.com/operator-framework/operator-sdk/releases/download/$RELEASE_VERSION/operator-sdk_linux_amd64
+        env:
+          RELEASE_VERSION: v1.26.0
```

Comment:
Same here. Why not to do two env vars instead and have it in global env for the workflow?